### DISCUSSION TITLE
Exporting Case information?

### DISCUSSION THREAD

**Discussion by drathbo** [Security-Onion-Solutions/securityonion#10101] [Unsupported Versions]
Hello. New to security onion and was playing around with the cases in the web ui. For audit and reporting purposes etc is it possible to export case information or at least case summaries (total count over a time, title, open close time, assignee etc)?

**Comment by weslambert** [comment]
You can generate a CSV report containing records from the `so-case` index from Discover inside of Kibana.

Additionally, you could create a dashboard inside of SOC to summarize the pertinent information.
